File: etc/kakao/i2021/ex1/Test1.java
package net.skhu.kakao.i2021.ex1;

public class Test1 {

    static class Solution {
        public int solution(String s) {
            String[] a = { "zero", "one", "two", "three", "four", "five", "six",
                           "seven", "eight", "nine" };
            for (int i = 0; i < a.length; ++i)
                s = s.replaceAll(a[i], String.valueOf(i));
            return Integer.valueOf(s);
        }
    }

    public static void main(String[] args) {
        Solution solution = new Solution();
        System.out.println(solution.solution("one4seveneight"));
        System.out.println(solution.solution("23four5six7"));
        System.out.println(solution.solution("2three45sixseven"));
        System.out.println(solution.solution("123"));
    }
}

File: baekjoon/b1003/test1.js
// 0, 1 출력 확인하기
let count
for (let i=0; i<=13; ++i) {
  count = [0,0]
  console.log("%d: %d, zero=%d, one=%d", i, fibonacci(i), count[0], count[1])
}

function fibonacci(n) {
  if (n==0) {
    ++count[0]
    return 0
  } else if (n==1) {
    ++count[1]
    return 1
  } else 
    return fibonacci(n-1) + fibonacci(n-2)
}